Rakefile

```diff
@@ -47,7 +47,6 @@ task :profile_memory do
   require 'memory_profiler'
 
   source = File.expand_path('lib/rouge/lexer.rb', __dir__)
-  sample = File.read(source, encoding: 'utf-8')
```

Comment:
@pyrmont Instead of deleting this line, please change `ln54` below to use `sample`

Reply:
@ashmaroli Done :)